--- libtunepimp-0.4.0/configure.in.orig	2005-11-12 00:43:39.000000000 +0100
+++ libtunepimp-0.4.0/configure.in	2005-11-17 01:11:21.416616000 +0100
@@ -65,14 +65,14 @@
 
 dnl Check for readline. This is optional.
 AC_CHECK_LIB(readline, readline,
-             [READLINE_LIBS="-lreadline -lncurses"
+             [READLINE_LIBS="-lreadline"
              AC_DEFINE(HAVE_LIBREADLINE,1,[Readline Support])],
              [echo "*"
              echo "*  The GNU readline library isn't installed."
              echo "*  This means the sample tagger, tp_tagger, won't"
              echo "*  have readline support."
              echo "*"
-	     ],-ltermcap)
+	     ])
 
 AC_CHECK_HEADERS(iconv.h)
 
