--- libtunepimp-0.5.0/plugins/tta/Makefile.am.orig	2006-07-01 03:07:17.000000000 +0200
+++ libtunepimp-0.5.0/plugins/tta/Makefile.am	2006-07-22 00:19:57.333111000 +0200
@@ -33,7 +33,7 @@
 all:
 	mkdir -p $(top_srcdir)/examples/plugins
 	rm -f $(top_srcdir)/examples/plugins/tta.tpp
-	ln -s ../../plugins/wv/.libs/tta.so $(top_srcdir)/examples/plugins/tta.tpp
+	ln -s ../../plugins/tta/.libs/tta.so $(top_srcdir)/examples/plugins/tta.tpp
 
 install-exec-am:
 
