diff -urN aa/libtunepimp-0.3.0/lib/threads/posix/thread.cpp libtunepimp-0.3.0/lib/threads/posix/thread.cpp
--- aa/libtunepimp-0.3.0/lib/threads/posix/thread.cpp	2004-01-11 10:27:52.000000000 +0100
+++ libtunepimp-0.3.0/lib/threads/posix/thread.cpp	2004-06-29 01:02:26.780296640 +0200
@@ -27,7 +27,8 @@
 #include <stdio.h>
 #include <errno.h>
 #include <string.h>
-#include <sched.h>                                                                                              
+#include <sched.h>
+#include <signal.h>
 #include "thread.h"
 
 #define DB printf("%s:%d\n", __FILE__, __LINE__);
